.github/workflows/envd-starship.yaml

```diff
@@ -0,0 +1,34 @@
+name: release
+
+on: 
+  release:
+    types: [created]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+
+  docker:
+    name: Push starship image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Docker Login
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERIO_USERNAME }}
+          password: ${{ secrets.DOCKERIO_TOKEN }}
+      - name: Docker Setup QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: build image
+        run: |
```

Comment:
Use build and push action instead. Example: https://github.com/tensorchord/envd-server/blob/main/.github/workflows/release.yml#L137-L146

Reply:
changed and tested. PTAL.